src/plugins/data/public/antlr/opensearch_sql/grammar/.antlr/OpenSearchSQLLexer.java

```diff
@@ -0,0 +1,2836 @@
+// Generated from /home/ubuntu/ws/OpenSearch-Dashboards/src/plugins/data/public/antlr/opensearch_sql/grammar/OpenSearchSQLLexer.g4 by ANTLR 4.13.1
+import org.antlr.v4.runtime.Lexer;
+import org.antlr.v4.runtime.CharStream;
+import org.antlr.v4.runtime.Token;
+import org.antlr.v4.runtime.TokenStream;
+import org.antlr.v4.runtime.*;
+import org.antlr.v4.runtime.atn.*;
+import org.antlr.v4.runtime.dfa.DFA;
+import org.antlr.v4.runtime.misc.*;
+
+@SuppressWarnings({"all", "warnings", "unchecked", "unused", "cast", "CheckReturnValue", "this-escape"})
+public class OpenSearchSQLLexer extends Lexer {
```

Comment:
For my knowledge, why we need this java file?

Reply:
these are auto-generated files from ANTLR engine for query parsing internally.